test/pricer_rule/utils.js

```diff
@@ -56,8 +56,10 @@ module.exports.createTokenEconomy = async (accountProvider, config = {}, eip20To
     organizationWorker,
   } = await Utils.createOrganization(accountProvider);
 
-  const tokenDecimals = eip20TokenConfig.decimals;
-  const token = await this.createEIP20Token(eip20TokenConfig);
+  const tokenDecimals = eip20TokenConfig.decimals || testConfig.decimals;
```

Comment:
Shouldn't the decimals be set on the `eip20TokenConfig` object instead? Wherever that is called? It appears likely that the configured value from ENV is now overridden by the tests calling this method.

Reply:
> Shouldn't the decimals be set on the eip20TokenConfig object instead?

@schemar: yes, I agree, it should. The issue here is that most calls to `createTokenEconomy` do not pass an `eip20TokenConfig` argument; only the `PricerRule.pay` tests do (and there it's a special case).

This solution seemed the most reasonable and also resolves a minor discrepancy in the existing testing (`tokenDecimals` =/= `token.decimals()` in the usual case).